lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -163,6 +163,7 @@ def add_card(post, card, action, options = {})
           post[:card][:network_token] = card.number
           post[:card][:cryptogram] = card.payment_cryptogram
           post[:card][:eci] = card.eci
+
           # used case of Network Token: 'CARD_ON_FILE', 'SUBSCRIPTION', 'UNSCHEDULED_CARD_ON_FILE'
```

Comment:
Sorry, I think I missed a few lines but we also need to move 167-176 to be inside like 182.